.taskcluster.yml

```diff
@@ -128,78 +123,97 @@ tasks:
                         git fetch ${repository} ${head_branch}
                         && git config advice.detachedHead false
                         && git checkout FETCH_HEAD
-                        && python automation/taskcluster/decision_task.py push
+                        && python automation/taskcluster/decision_task.py pull-request
+                    env:
+                      GITHUB_PULL_TITLE: ${pull_request_title}
                   extra:
                     treeherder:
-                      symbol: D
+                      symbol: D-PR
                   metadata:
-                    name: Fenix - Decision task
-                    description: Schedules the build and test tasks for Fenix.
-        else:
-          - $if: 'tasks_for == "cron"'
-            then:
-              $mergeDeep:
-                - {$eval: 'default_task_definition'}
-                - scopes:
-                    - $if: 'trust_level == 3'
-                      then: assume:hook-id:project-mobile/fenix-nightly
-                      else: assume:hook-id:project-mobile/fenix-nightly-staging
-                  routes:
-                    - notify.email.fenix-eng-notifications@mozilla.com.on-failed
-                  payload:
+                    name: 'Fenix - Decision task (Pull Request #${pull_request_number})'
+                    description: 'Building and testing the Fenix - triggered by [#${pull_request_number}](${pull_request_url})'
+        - $if: 'tasks_for == "github-push" && head_branch[:10] != "refs/tags/"'
+          then:
+            $mergeDeep:
+              - {$eval: 'default_task_definition'}
+              - scopes:
+                  - ${assume_scope_prefix}:branch:${short_head_branch}
+                payload:
+                  command:
+                    - >-
+                      git fetch ${repository} ${head_branch}
+                      && git config advice.detachedHead false
+                      && git checkout FETCH_HEAD
+                      && python automation/taskcluster/decision_task.py push
+                extra:
+                  treeherder:
+                    symbol: D
+                metadata:
+                  name: Fenix - Decision task
+                  description: Schedules the build and test tasks for Fenix.
+        - $if: 'tasks_for == "github-release"'
+          then:
+            $mergeDeep:
+              - {$eval: 'default_task_definition'}
+              - scopes:
+                - ${assume_scope_prefix}:release
+                payload:
+                  command:
+                    - >-
+                      git fetch ${repository} refs/tags/${head_rev}
+                      && git config advice.detachedHead false
+                      && git checkout FETCH_HEAD
+                      && python automation/taskcluster/decision_task.py beta ${event.release.tag_name}
+                extra:
+                  treeherder:
+                    symbol: B
```

Comment:
Nit: `B` might be confused with "build", for people who come from mozilla-central. We might want to go with `beta`. I'm fine changing `N`, for the sake of consistency.

Reply:
Good point!
> I'm fine changing N, for the sake of consistency.

As in changing `N` to `nightly`, right?

Reply:
Right :)